eventbus/common/interface.go

```diff
@@ -42,5 +42,9 @@ type EventSourceDriver interface {
 
 type SensorDriver interface {
 	Initialize() error
-	Connect(triggerName string, dependencyExpression string, deps []Dependency) (TriggerConnection, error)
+	Connect(ctx context.Context,
```

Comment:
Added context and atLeastOnce boolean to this interface, the stan and jetstream implementation just ignore these values